storybook/src/foundation/design-tokens/space.docs.mdx

```diff
@@ -74,28 +74,21 @@ In Compact Mode, the medium grid space grows from 16 to 40 pixels between window
 <table>
```

Comment:
Why was this HTML again?

Reply:
Because we used `colspan`, but as of this PR we no longer do :) Converted back to Markdown.